mirar/processors/photometry/utils.py

```diff
@@ -251,10 +251,13 @@ def aper_photometry(
     # error = calc_total_error(data, bkg_error, effective_gain)
     # phot_table = aperture_photometry(diff_cutout - bkg, aperture, error=error)
     # counts_err = phot_table['aperture_sum_err'][0]
-    error = np.sqrt(np.sum(aperture_unc_data**2))
+    error = np.sqrt(np.nansum(aperture_unc_data**2.0))
```

Comment:
how was this returning a negative value?

Reply:
No idea. I don't think I fixed it, so I'm leaving the assertion error>0 check in there.

Reply:
The above change just fixes nan error

Reply:
I found the place, will fix it in my PR.